YCAI/webpack.config.js

```diff
@@ -127,6 +127,19 @@ if (mode === 'production') {
   );
 }
 
+if (mode === 'development') {
+  plugins.push(
+    new WebpackPluginServe({
```

Comment:
you're not using the native webpack dev server? https://webpack.js.org/configuration/dev-server/